src/service/__specs__/mobbers.spec.js

```diff
@@ -16,6 +16,27 @@ describe('state/mobbers', () => {
     })
   })
 
+  describe('#shuffleMobbers', () => {
+    it('shuffles the mobbers so there is a different order', () => {
+      subject.addMobber(mobber);
+      subject.addMobber( {name: 'TestersonFace', id: 'mobber-2'});
+      subject.addMobber( {name: 'TestersonHead', id: 'mobber-3'});
+      subject.addMobber( {name: 'TestersonNose', id: 'mobber-4'});
+      let mobberCopy = subject.getAll().slice();
+      subject.shuffleMobbers();
+      let areSameArray = true;
+      for(let i = 0; i < mobberCopy.length; i++)
+      {
+        if(mobberCopy[i] !== subject.getAll()[i])
+        {
+          areSameArray = false;
+          break;
+        }
+      }
+      expect(areSameArray).equal(false);
```

Comment:
Feels like this check is more complicated than it needs to be. Why not do a deep equality comparison, or at least do a map & join then check for string equality? The for loop logic makes the test code less readable.

Reply:
Also, the implementation is using Math.random(), so isn't there a chance that you'll get the same array order back? Would that cause transient failures in the test? If so, maybe we should shuffle multiple times and make sure that most of them don't end up the same?

Alternatively, you could abstract or hijack (I think that sinon can do this) the random number generation to make it deterministic for your test.